SpatialGDK/Source/SpatialGDK/Private/EngineClasses/SpatialNetDriverDebugContext.cpp

```diff
@@ -364,7 +364,7 @@ void USpatialNetDriverDebugContext::TickServer()
 				// There is a requirement of readiness before we can use SendAddComponent
 				if (IsActorReady(Actor))
 				{
-					Worker_ComponentData CompData = View.Component.CreateDebugComponent();
+					Worker_ComponentData CompData = View.Component.CreateComponentData();
```

Comment:
Commenting here, because it's the first instance of this change.
But not sure why we did it this way.
Maybe to avoid dynamic dispatch or something?

Reply:
Unexpected poetry

Reply:
I seem to remember it was so we could template similar functionality.